pkg/scheduler/eventhandlers.go

```diff
@@ -430,8 +430,7 @@ func addAllEventHandlers(
 		)
 	}
 
-	// On add and delete of PVs, it will affect equivalence cache items
-	// related to persistent volume
+	// On add and delete of PVs.
```

Comment:
Isn't it update?

Reply:
Oh right, got confused sorry
/lgtm cancel

Reply:
It has been updated.

Reply:
@lixiaobing1 this line still says `On add and delete of PVs`, it should say `On add and update of PVs`. Sorry for the confusion

Reply:
It's my mistake, has been modify to: On add and update of PVs.